PyTorch/SpeechSynthesis/FastPitch/fastpitch/data_function.py

```diff
@@ -206,12 +208,21 @@ def __init__(self,
 
     def __getitem__(self, index):
         # Separate filename and text
-        if self.n_speakers > 1:
+        if self.n_speakers > 1 and self.n_conditions < 1:
```

Comment:
this block is a little cumbersome but I really appreciate how legible it is *appreciation comment*